src/components/Picture/Picture.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box, HTMLChakraProps, forwardRef, Image } from "@chakra-ui/react";
+
+export interface PictureProps extends HTMLChakraProps<"img"> {
```

Comment:
This extends the "img" b/c we want to forward styles to the `<Image />`. We still pass the ref to the root element, which renders a `<picture />`